### ISSUE TITLE
Renommer le dépôt

### ISSUE THREAD

**Issue by Olon11** [profcfuhrmanets/log210-jeu-de-des#150]
Il n'est pas nécessaire d'avoir `node-express-ts` dans le nom du dépôt. Je renommerais le dépôt `log210-jeu-de-des`.

**Comment by fuhrmanator** [comment]
Je suis d'accord avec ton point. Par contre, renommer provoque potentiellement des changements dans beaucoup de documentation.

Vu que GitHub Classroom va nous obliger d'avoir les dépôts dans la même organisation que la Classroom, on pourra faire ces changements (renommage, correction de liens) ensemble. https://github.com/orgs/community/discussions/127455/

**Comment by Olon11** [comment]
J'ai renommé le dépôt et mis à jour les liens dans l'énoncé.
